packages/core/src/App/Containers/RealAccountSignup/add-currency.jsx

```diff
@@ -227,6 +227,10 @@ AddCurrency.propTypes = {
     openRealAccountSignup: PropTypes.func,
     setShouldShowAllAvailableCurrencies: PropTypes.func,
     deposit_target: PropTypes.string,
+    onSubmit: PropTypes.func,
+    is_add_crypto: PropTypes.bool,
+    is_add_fiat: PropTypes.bool,
+    hasNoAvailableCrypto: PropTypes.bool,
```

Comment:
I think the type of this is a function, as it is used like this in the file:
```
'account-wizard--disabled': hasNoAvailableCrypto(),
```

Reply:
Changed it.Thanks